README.md

```diff
@@ -154,6 +154,64 @@ python -m gcp_variant_transforms.vcf_to_bq \
   --runner DataflowRunner
 ```
 
+### Security and Compliance
+
+You may need to constrain Cloud Dataflow job processing to a specific
+geographic region in support of yourproject’s security and compliance needs.
+For example, the [European Union Data Protection Directive](https://cloud.google.com/security/compliance/eu-data-protection-directive/).
+
+For this, you need to specify zone and region in Dataflow API (as well as
+Pipeline API if you are using Docker). For example, in order to restrict
+job processing to Europe. First update `vcf_to_bigquery.yaml` to use zone and
+region in Europe
+
+```yaml
+name: vcf-to-bigquery-pipeline
+docker:
+  imageName: gcr.io/gcp-variant-transforms/gcp-variant-transforms
+  cmd: |
+    ./opt/gcp_variant_transforms/bin/vcf_to_bq \
+      --project my_project \
+      --input_pattern gs://my_bucket/vcffiles/*.vcf \
+      --output_table my_project:my_bigquery_dataset.my_bigquery_table \
+      --staging_location gs://my_bucket/staging \
+      --temp_location gs://my_bucket/temp \
+      --job_name vcf-to-bigquery \
+      --runner DataflowRunner
+      --region europe-west1 \
```

Comment:
Can we make the new fields bolded inside the doc? Otherwise, I suggest doing something like [dsub](https://github.com/googlegenomics/dsub/blob/master/docs/code.md#--command-shell-string) where they use `...` and only emphasize the new args that need to be added.  ditto below.

Reply:
Done!